tasks/gulp/copy-to-destination.js

```diff
@@ -15,54 +15,61 @@ const taskArguments = require('../task-arguments')
 
 gulp.task('copy-files', () => {
   return merge(
+    // Copy source JavaScript
```

Comment:
```suggestion
    // Copy ESM files to make them available in the package
```

Reply:
Can I suggest a tweak to align with the Gulpfile task?
https://github.com/alphagov/govuk-frontend/blob/542e7ec7bd2d047b19313d35fd9ba17e32cef8ac/gulpfile.js#L28

At the moment we use `--destination` to choose **./package** so the destination could vary

```suggestion
    // Copy ESM JavaScript to destination, ready for publishing
```